day_two_guide/topics/adding_node_host.adoc

```diff
@@ -234,5 +235,6 @@ node "node04" labeled
 $ *oc label node node04 test-*
 node "node04" labeled
 ----
+////
```

Comment:
@bfallonf FYI, I have commented this bit out for now because it needs re-addressing since we can't use `openshift_node_labels` anymore. I have an idea how we can update it in a follow-up, but removing it temporarily.